loky/backend/context.py

```diff
@@ -219,39 +167,43 @@ def _count_physical_cores():
     try:
         if sys.platform == "linux":
             cpu_info = subprocess.run(
-                "lscpu --parse=core".split(" "), capture_output=True)
-            cpu_info = cpu_info.stdout.decode("utf-8").splitlines()
+                "lscpu --parse=core".split(" "),
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                universal_newlines=True)
```

Comment:
On Python 3.6 the previous version sometimes fails, as `capture_output` argument has been added to `subprocess.run` only [since Python 3.7](https://docs.python.org/3/library/subprocess.html#subprocess.run).

Reply:
Since Python 3.6 is no longer supported by the CPython developers, I wouldn't mind dropping support in loky at the same time.

Reply:
Then I'll drop Python 3.6 too.

Reply:
Fixed by manual installation of PyPy 3.7